nucypher/policy/policies.py

```diff
@@ -512,8 +512,8 @@ def enact(self, network_middleware, publish_treasure_map=True) -> dict:
             if publish_treasure_map is True:
                 return self.publish_treasure_map(network_middleware=network_middleware)  # TODO: blockchain_signer?
 
-    def propose_arrangement(self, network_middleware, ursula, arrangement) -> bool:
-        negotiation_response = network_middleware.propose_arrangement(arrangement=arrangement)
+    def propose_arrangement(self, network_middleware, arrangement) -> bool:
+        negotiation_response = network_middleware.propose_arrangement(arrangement=arrangement)  # Wow, we aren't even passing node here.
```

Comment:
is that approval or disapproval :sweat_smile: ?

Reply:
hahaha; in these crazy times, they feel exactly the same